packages/lab/src/Table/renderers/DropdownColumnCell/DropdownColumnCell.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { HvDropdown } from "@hitachivantara/uikit-react-core";
+
+const HvDropdownColumnCell = ({ value, disabled, onChange, placeholder, dropdownProps }) => {
+  return (
+    <HvDropdown
+      onChange={(item) => {
+        onChange?.(item);
+      }}
+      placeholder={placeholder}
+      disabled={disabled}
+      values={value}
+      disablePortal
+      {...dropdownProps}
+    />
+  );
+};
+
+HvDropdownColumnCell.propTypes = {
+  /**
+   * values to render in the dropdown.
+   */
+  value: PropTypes.arrayOf(
```

Comment:
We could use the same props from the dropdown, values. Or this this intentional?